src/main/kotlin/no/nav/dagpenger/regel/minsteinntekt/InngangsvilkårSpesifikasjoner.kt

```diff
@@ -25,13 +25,13 @@ internal val kravTilMinsteinntekt: Spesifikasjon<Fakta> = (ordinær eller vernep
         beskrivelse = "Krav til minsteinntekt"
     )
 
-internal val kravTilMinsteinntektKorona: Spesifikasjon<Fakta> = (koronaOrdinær eller verneplikt)
+internal val kravTilMinsteinntektKorona: Spesifikasjon<Fakta> = (koronaOrdinær eller verneplikt eller lærling)
     .med(
         identifikator = "Krav til minsteinntekt",
         beskrivelse = "Krav til minsteinntekt"
     )
 
 fun Evaluering.koronaRegelBrukt() =
-    if (this.children.any { it.identifikator == "Krav til minsteinntekt etter midlertidig korona-endret § 4-4" })
+    if (this.children.any { it.identifikator == koronaOrdinær.identifikator || it.identifikator == lærling.identifikator })
```

Comment:
Skal vi bruke KORONA-beregningsregelen slik den er? SKal vi kanskje endre i sats også da?

Reply:
jeg er veldig usikker om hva disse er brukt til kjenner jeg, det er primært for Arena ikke sant?